NEWS.rst

```diff
@@ -1,6 +1,16 @@
 History
 =======
 
+0.12 (2019-02-12)
+-----------------
+* Make L1 product by applying calibration corrections aka applycal (#194 - #198)
+* Let default reference antenna in v4 datasets be "array" antenna (#202, #220)
+* Use katsdptelstate v0.7 features: msgpack, memory backend (#196, #201, #212)
```

Comment:
We're not using msgpack anywhere, just using the encode/decode helpers.

Reply:
Sure.